README.md

```diff
@@ -0,0 +1,49 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="images/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Hello Microverse</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+- [👥 Authors](#authors)
+- [📝 License](#license)
```

Comment:
- [ ] Please kindly fill other sections of the README file, we add a README file to a repository to communicate important information about the project: why the project is useful; what people can do with the project; and how they can use it. The sections listed below are very vital. 👍 

> 🛠 Built With,  Key Features,  Getting Started, 🔭 Future Features